health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/AppModule.js

```
@@ -84,207 +85,217 @@ const AppModule = () => {
-                  style={{
-                    position: "absolute",
-                    left: "-10px",
-                    top: "-14px",
-                  }}
+      <EqualHeightWrapper deps={[modulesData]}>
+        <div className="modules-container">
+          {modulesData
+            ?.sort((x, y) => x?.data?.order - y?.data?.order)
+            // ?.filter((module) => module?.data?.isDisabled === "false")
+            .map((module, index) => (
+              <Card className={`module-card ${selectedModuleCodes.includes(module?.data?.name) ? "selected-card" : ""}`}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)